modules/ROOT/pages/business-groups.adoc

```diff
@@ -0,0 +1,102 @@
+
+=== Create a Business Group
+
+Create business groups within your organization. 
+
+[NOTE]
+There is a limit of 100 business groups per organization. 
+
+. In the Anypoint Platform *Access Management* page, in the left menu, click *Organization*.
+. Click the blue plus sign next to the organization name or any of the business groups in the tree diagram.
++
+image::organization-20885.png[]
++
+. In the dialog box, enter:
+  * *Business Group name*: Name for the new business group.
+  * *Owner*: Assign an existing user in the organization to be the business group owner.
+  * Select *Owner can create Business Groups* to allow the owner to create business groups that branch under this business group.
+  * Select *Owner can create environments* to allow the owner to create environments within this business group.
+  * You can assign some or all of the vCores that your organization owns to an individual business group. This ensures that these are used by the CloudHub deployments within the business group. You can assign these when creating the business group or edit these settings later.
+  * *Static IPs*: Enable the use of static IPs.
+. Click *Add Business Group*.
+
+image::organization-9fa5f.png[]
+
+[NOTE]
+--
+Allocating vCores to a business group makes those vCores available only to that business group, which makes them unavailable to the parent organization.
+--
+
+==== Child Business Groups
+
+You can create hierarchical levels of business groups to provide more control over access. To create a child business group click the blue *+* icon next to any business group.
+
+image::organization-2b25c.png[]
+
+Since business groups are hierarchical, the owner of a parent business group always retains administrator permissions for any child business group within that parent. This means that the owner of a business group can create child business groups and assign another user as the owner (owners automatically have the *Organization Administrators* role) of those child business groups. The owner of the parent group remains an *Organization Administrator* within the child business group. 
+
+Conversely, owners of child business groups cannot:
+
+* Access or modify the parent business group or master organization settings
+* View the parent business group's client ID and client secret
+
+When you create a child business group within a parent, only the vCores that were assigned to the parent business group can be allocated to the child.
+
+=== Navigating Between Business Groups
+
+When your organization has multiple business groups, you can navigate between them using the menu at the top-right corner. Switching between business groups changes the list of available CloudHub deployments, APIs, users, and roles settings.
+
+image::switch-suborg.png[switch+suborg]
+
+[NOTE]
+--
+When you sign into Anypoint Platform, you are signed back into the organization or business group you were in when you last signed out.
```

Comment:
how does that work? If I am a member of a biz group, am I always signed back into it? is the master org also a "master" biz group if that's enabled, even if I haven't created any biz groups yet?

Reply:
Also, I like the caveat you include with this in organizations.adoc: "always verify you are in the correct organization or business group."

Reply:
Let's reword so that you're not "signed into" a BG, but rather have an active one. You're always signed into your user account, not an organization.